tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_homescreen_change_wallpaper.py

```diff
@@ -18,13 +18,14 @@ def test_homescreen_change_wallpaper_from_gallery(self):
         """
         https://moztrap.mozilla.org/manage/case/1902/
         """
-
-        homescreen = Homescreen(self.marionette)
         self.apps.switch_to_displayed_app()
-
         default_wallpaper_settings = self.data_layer.get_setting('wallpaper.image')
-        contextmenu = homescreen.open_context_menu()
-        activities = contextmenu.tap_change_wallpaper()
+
+        # open settings app and choose to change wallpaper, instead of opening context menu
+        settings = Settings(self.marionette)
+        settings.launch()
+        homescreen_page = settings.open_homescreen()
+        activities = homescreen_page.pick_wallpaper()
```

Comment:
hmm, that script compares system.wallpaper_properties value, while this one compares self.data_layer.get_setting('wallpaper.image').  are they equivalent?  If so I'll remove the script